pkg/templates/nstemplatetiers/nstemplatetier_generator_whitebox_test.go

```diff
@@ -96,7 +96,7 @@ func TestParseAllRevisions(t *testing.T) {
 		assert.Equal(t, "123456c", revisions["advanced"]["stage"])
 		assert.Equal(t, "123456d", revisions["basic"]["code"])
 		assert.Equal(t, "123456e", revisions["basic"]["dev"])
-		assert.Equal(t, "123456f", revisions["basic"]["stage"])
+		assert.Equal(t, "1234567", revisions["basic"]["stage"])
```

Comment:
As I understood, commit_hash is string, it can be `alpha-numeric` or `numeric-only`.  Why do we need to handle commit_hash `numeric-only` in some special way ??

Reply:
because the YAML parser will convert the value into an integer if it can, resulting in an error since the we expect a `string` in https://github.com/codeready-toolchain/host-operator/pull/86/files/4d2b183c98d113de5ba58161cd4c864c2e89c446#diff-6d635ca031a9d241865459f7ef30f167L112